bokeh/io.py

```diff
@@ -358,11 +357,10 @@ def save(obj, filename=None, resources=None, title=None, state=None, validate=Tr
     ``/foo/myplot.html``)
 
     Args:
-        obj (Document or model object) : a plot object to save
+        obj (LayoutDOM object) : a Layout (Row/Column), Plot or Widget object to display
 
         filename (str, optional) : filename to save document under (default: None)
-            If None, use the default state configuration, otherwise raise a
-            ``RuntimeError``.
```

Comment:
This docstring is out-of-date. The implementation now will create a tempfile matching the `__file__` name instead of raising a RuntimeError.